pom.xml

```diff
@@ -3965,6 +3977,46 @@
               </exclusion>
             </exclusions>
           </dependency>
+          <dependency>
```

Comment:
The dependence on Hadoop was tricky. Ensure that the dependency tree does not have a new dependency.

Reply:
My audit of jars that were not present in 1.20.1-hadoop2

#### Comes from Google Sheets plugin
Only in /opt/apache-drill/jars/3rdparty: opencensus-api-0.31.1.jar
Only in /opt/apache-drill/jars/3rdparty: opencensus-contrib-http-util-0.31.1.jar
Only in /opt/apache-drill/jars/3rdparty: grpc-context-1.27.2.jar
Only in /opt/apache-drill/jars/3rdparty: google-api-client-1.35.2.jar
Only in /opt/apache-drill/jars/3rdparty: google-api-client-jackson2-1.35.2.jar
Only in /opt/apache-drill/jars/3rdparty: google-api-services-sheets-v4-rev20220606-1.32.1.jar
Only in /opt/apache-drill/jars/3rdparty: google-auth-library-credentials-1.8.0.jar
Only in /opt/apache-drill/jars/3rdparty: google-auth-library-oauth2-http-1.8.0.jar
Only in /opt/apache-drill/jars/3rdparty: google-http-client-1.42.0.jar
Only in /opt/apache-drill/jars/3rdparty: google-http-client-apache-v2-1.42.0.jar
Only in /opt/apache-drill/jars/3rdparty: google-http-client-gson-1.42.0.jar
Only in /opt/apache-drill/jars/3rdparty: google-oauth-client-1.34.1.jar
Only in /opt/apache-drill/jars/3rdparty: google-oauth-client-java6-1.34.1.jar
Only in /opt/apache-drill/jars/3rdparty: google-oauth-client-jetty-1.34.1.jar

#### Comes from Swagger
Only in /opt/apache-drill/jars/3rdparty: narcissus-1.0.1.jar
Only in /opt/apache-drill/jars/3rdparty: jvm-driver-4.0.0.jar
Only in /opt/apache-drill/jars/3rdparty: classgraph-4.8.117.jar
Only in /opt/apache-drill/jars/3rdparty: swagger-annotations-2.1.12.jar
Only in /opt/apache-drill/jars/3rdparty: swagger-core-2.1.12.jar
Only in /opt/apache-drill/jars/3rdparty: swagger-integration-2.1.12.jar
Only in /opt/apache-drill/jars/3rdparty: swagger-jaxrs2-2.1.12.jar
Only in /opt/apache-drill/jars/3rdparty: swagger-jaxrs2-servlet-initializer-v2-2.1.12.jar
Only in /opt/apache-drill/jars/3rdparty: swagger-models-2.1.12.jar

#### Looks like nimbus-jose-jwt was downgraded in Hadoop 2, causing this downgrade
Only in /opt/apache-drill/jars/3rdparty: accessors-smart-1.2.jar